### PR TITLE
Include (add) the imgproxy docker image for dynamic image resizing ++

### PR DESCRIPTION
**Pull request by gpomer**
https://higherlogic.atlassian.net/browse/VNLA-1263

Add the Docker image for "imgproxy" software library for providing on-the-fly dynamic image resizing to provide image-sets from single src image resources. 

Added anSpecific Addon with the the interface for implementing imgproxy to generate the source-set (multiple sizes) urls from the available single image.

To install in your local docker you can simply _rebuild your docker_ containers and you will have the new image responding at http://localhost:8600.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -8,3 +8,10 @@ VANILLA_DOCKER_FASTCGI_BUFFER_SIZE=128k
 VANILLA_DOCKER_FASTCGI_BUFFERS="4 256k"
 VANILLA_DOCKER_FASTCGI_BUSY_BUFFERS_SIZE=256k
 VANILLA_DOCKER_FASTCGI_TEMP_FILE_WRITE_SIZE=256k
+
+# https://docs.imgproxy.net/configuration
+IMGPROXY_IGNORE_SSL_VERIFICATION=true
+#https://docs.imgproxy.net/signing_the_url
+#IMGPROXY_KEY=
+#IMGPROXY_SALT=
+#IMGPROXY_SIGNATURE=
```
Comment (vanilla-dbarbier) [inline .env:17]: Are those meant to be used in the future?

**Review by vanilla-dbarbier — CHANGES_REQUESTED** [review]
This _looks_ alright.
However, are there means for me to test this on my local?

File: .env (hunk 1 above)
Comment (gpomer) [reply .env:17]: yes these would be enabled in production for signing the requests, so no external entities could use our imgproxy install if they didn't have the key. They are there so people can see where to put them if they want to test/work with signatures. They do work, but they have to also be added to the config file in Vanilla to generate the signature for the requests.

Comment (gpomer) [reply .env:17]: To install in your local docker you can pull the change (this branch) _rebuild your docker_ containers and you will have the new image responding at http://localhost:8600.

**Review by vanilla-dbarbier — APPROVED** [review]
Tested as functional.
Approved!

**Pull request merged**